src/fakeData/facilities.ts

```diff
@@ -0,0 +1,26 @@
+import { Facility } from '../typeDefs/gqlTypes'
+
+export const fakeFacilities = () => {
+    const facility : Facility = {
+        nameEn: 'Zoo',
+        nameJa: '動物園',
+        contact: {
+            address: {
+                // generate fake data from type PhysicalAddress in gqlTypes.ts file
+                addressLine1En: '1-1-1',
+                addressLine2En: 'Ueno',
+                cityEn: 'Taito',
+                prefectureEn: 'Tokyo',
+                postalCode: '100-0000'            
+            },
+            email: 'zoo@test.com',
+            phone: '08000000000',
+            website: 'https://zoo.test.com',
+            mapsLink: ''
+        },
+        healthcareProfessionalIds: [],
+        healthcareProfessionals: []
+    }
+
+    return [facility]
+}
```

Comment:
This file was removed in #227 because it wasn't being used. Is there a reason we need to reintroduce the fake data?

Reply:
yes, we want fake data. 
1) this will be used for when we want to test endpoints by adding data, updating it, then deleting it.
2) we will also want seed data for when we load up a fake/test database so the site isn't empty. 

Keep in mind this is just a starting point and we'll need to update this iteratively.